crates/re_log/src/setup.rs

```diff
@@ -60,3 +66,45 @@ pub fn setup_web_logging() {
     )))
     .expect("Failed to install logger");
 }
+
+// ----------------------------------------------------------------------------
+
+#[cfg(not(target_arch = "wasm32"))]
+fn env_var_bool(name: &str) -> Option<bool> {
```

Comment:
I could swear I've seen this snippet several times over in the codebase, but can't find other instances right now